make/config.mk

```diff
@@ -48,6 +48,15 @@ USE_CUDA_PATH = NONE
 # whether use CuDNN R3 library
 USE_CUDNN = 0
 
+# CUDA architecture setting: going with all of them.
+# For CUDA < 6.0, comment the *_50 lines for compatibility.
+CUDA_ARCH := -gencode arch=compute_20,code=sm_20 \
```

Comment:
@piiswrong I think you are confused by the comments.

arch is controlled what hardware the code generate for. this is like x86/x64. CUDA version is like libc version. CUDA 6.0 doesn't support _50, but still support other arch.